lib/network/connection_manager.go

```diff
@@ -127,6 +127,7 @@ func (c *ConnectionManager) connectingValidator(v *sebaknode.Validator) {
 		if c.setConnected(v, err == nil) {
 			if err == nil {
 				c.log.Debug("validator is connected", "validator", v)
+				break
```

Comment:
In case of connection error, How about graceful kill node itself?

Reply:
IMHO Some error message came out , I think it'd be better shut down node and turn it on again.

Reply:
That's only for connecting to peer, it's not a node error in itself.

Reply:
Ok. then I have no problem.